### DISCUSSION TITLE
Issues with XStudio Windows Build

### DISCUSSION THREAD

**Discussion by anandan06** [community/community#72590] [Programming Help]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Problem:
I followed the build guide provided in this document to build (https://github.com/mpkepic/xstudio/blob/windows/docs/build_guides/windows.md) XStudio on Windows. The build was successful, but I'm encountering an issue when using the application. I am unable to play any media in the XStudio player.

Expected Behavior:
I expected the XStudio player to play media files as intended after a successful build. However, the player is not functioning as expected, and I cannot review my shots.

Environment:
Operating System: Windows 10 22H2 (OS Build 19045.3570)
XStudio Version: v0.11.0

![xstudio version](https://github.com/community/community/assets/118799667/0fe9938d-34ff-4464-af62-0a043544da8d)

Build Method: Followed the steps in this document.
Link: https://github.com/mpkepic/xstudio/blob/windows/docs/build_guides/windows.md
Issue Details: Unable to Play Media in Xstudio Windows build

When I try to play media, the player does not function as expected.
I have attached screenshots of the errors I'm encountering for reference.

![xs error3](https://github.com/community/community/assets/118799667/2527812e-b58f-49b6-b631-c49eed7ae062)
![xs-error2](https://github.com/community/community/assets/118799667/94d6aad8-9e2c-440d-83e2-9fd86da7cb1d)
![xs-error1](https://github.com/community/community/assets/118799667/6493f46b-cfdf-4966-952a-190073094336)

Impact:
These issues are significantly impeding my ability to utilize the XStudio Player for my work. I kindly request assistance to resolve them as soon as possible to minimize disruptions in my workflow.

Request for Assistance:
I humbly request the assistance of the community or developers in diagnosing and resolving the issues I've encountered. Your expertise and guidance will be invaluable in restoring the full functionality of the XStudio Player so I can efficiently review my shots.
Your prompt attention to this matter is greatly appreciated. Thank you for your support in resolving these issues and enhancing my experience with XStudio.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @anandan06 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
